packages/connect-thegraph-token-manager/subgraph/src/TokenManager.ts

```diff
@@ -1,42 +1,30 @@
 import { Address } from '@graphprotocol/graph-ts'
 import { TokenManager as TokenManagerContract } from '../generated/templates/TokenManager/TokenManager'
 import { TokenManager as TokenManagerEntity } from '../generated/schema'
-import { createMiniMeTokenTemplateAndEntity } from './MiniMeToken'
 // import { NewVesting as NewVestingEvent } from '../generated/templates/TokenManager/TokenManager'
```

Comment:
Do you leave this commented to improve the sync time? Should we include a comment about that?

Reply:
Yep, that's the point. I'll add a comment!